src/main/webapp/WEB-INF/view/profile.jsp

```diff
@@ -1,28 +1,53 @@
 <%@ page import="codeu.model.data.User" %>
 <%@ page import="codeu.model.store.basic.UserStore" %>
+<%@ page import="org.kefirsf.bb.BBProcessorFactory" %>
+<%@ page import="org.kefirsf.bb.TextProcessor" %>
+
 
 <%
-User user = (User) request.getAttribute("user");
+User user = (User) request.getAttribute("userToAccess");
+String loggedInUserename = (String) request.getSession().getAttribute("user");
+User loggedInUser = UserStore.getInstance().getUser(loggedInUserename);
+// processor needed for BBCode to HTML translation
+TextProcessor processor = BBProcessorFactory.getInstance().createFromResource("kefirbb.xml");
 %>
 
 <!DOCTYPE html>
 <html>
 <head>
-  <title>Profile Page</title>
+  <meta charset="UTF-8">
+  <title><%= user.getName() %>'s Profile Page</title>
   <link rel="stylesheet" href="/css/main.css">
 </head>
 <body>
 
   <%@ include file="/WEB-INF/view/navbar.jsp" %>
 
   <div id="container">
-    <div
-      style="width:75%; margin-left:auto; margin-right:auto; margin-top: 50px;">
-      <h1><%= user.getName()%>'s profile!</h1>
+
+    <h1><%= user.getName() %>'s Profile Page</h1>
+
+    <hr/>
+
+      <h2>About <%= user.getName()%></h2>
       <p>
-        This is a plain profile page!
+        <%= processor.process(user.getAboutMe())%>
       </p>
-    </div>
+
+    <% if (loggedInUser != null && loggedInUser.getName().equalsIgnoreCase(user.getName())) { %>
```

Comment:
I'd use equals() and not equalsIgnoreCase() because our app allows for these users to exist: user, User, uSer, usEr, etc.